src/platform/linux/installing.md

```diff
@@ -2,7 +2,48 @@
 
 <!-- toc -->
 
-## Requirements
+It's possible to install Anki on Linux using a variety of methods. The easiest way is to use the official packages, but you can also install Anki from source as shown below.
+
+## Flatpak
+
+If your distribution supports [Flatpak](https://flatpak.org/setup/), you can install Anki from the official [Flathub](https://flathub.org/) repository:
```

Comment:
Users may potentially think you're saying the Anki FlatPak is official; please remove that word.